ci/setup.py

```diff
@@ -0,0 +1,16 @@
+from setuptools import find_packages, setup
+
+setup(
+    name="ray_ci",
+    packages=find_packages(
+        where=".",
+        include=[
+            package for package in find_packages() if package.startswith("ray_ci")
+        ],
+    ),
+    version="0.0.1",
+    author="Ray Team",
+    description="The Ray OSS ci testing package",
+    url="https://github.com/ray-project/ray",
+    install_requires=["numpy"],
+)
```

Comment:
how about I convert this into hermetic bazel for you, so that you do not need this `setup.py` thing?

Reply:
Absolutely ;)